.github/workflows/publish_testpypi.yml

```
@@ -9,18 +9,22 @@ on:
 jobs:
   super-test:
     strategy:
-      max-parallel: 2
+      fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python_v: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13', '']
+        python_v: ['3.8', '3.9', '3.10', '3.12', '3.13', '3.14', '3.14t']
```

```suggestion
        python_v: ['3.8', '3.9', '3.10', '3.12', '3.13', '3.14']
```
@ayjayt Let's drop `3.14t` from the CI until it runs successfully. (Or mark as expected to fail, if GHA provides a way to do that.) Keep the CI green.